ES/ESmy012.xml

```diff
@@ -867,7 +858,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                        <placeName ref="LOC7347Aymaat">ʾAddi ʾAymaʿāt Mikāʾel</placeName>.
                      </origin>
```

Comment:
I do not quite follow why the split between `origin `and `acquisition`
Since acquisition is a separate tag it is not quite OK to start with "It" - if we distribute the display differently it is not clear what "It" refers to. I would put the chain of events under `provenance`

Reply:
I rephrased, but this is a complex case conserving several MY manuscripts, to be explaind summarily elsewhere.